### PR TITLE
Use {{page.version}} instead of hard-coded release number

### PR DESCRIPTION
**Pull request by janetkuo**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2238)

### CODE REVIEW

**Comment by jaredbhatti** [comment]
Can you fix the conflicts?

File: _data/guides.yml [hunk 1]
```diff
@@ -95,7 +95,7 @@ toc:
   - docs/user-guide/connecting-to-applications-proxy.md
   - docs/user-guide/connecting-to-applications-port-forward.md
   - title: Using Explorer to Examine the Runtime Environment
-    path: https://github.com/kubernetes/kubernetes/tree/release-1.3/examples/explorer
```
Comment (janetkuo) [inline _data/guides.yml]: We can't use {{page.version}} in .yml so hard-coded

File: _data/samples.yml [hunk 2]
```diff
@@ -6,72 +6,70 @@ toc:
...
-    path: https://github.com/kubernetes/kubernetes/tree/release-1.3/examples/spark
+    path: https://github.com/kubernetes/kubernetes/tree/release-1.5/examples/spark
   - title: Apache Storm
-    path: https://github.com/kubernetes/kubernetes/tree/release-1.3/examples/storm
+    path: https://github.com/kubernetes/kubernetes/tree/release-1.5/examples/storm
 
 - title: Messaging / Queueing
   section:
-  - title: Celery + RabbitMQ
-    path: https://github.com/kubernetes/kubernetes/tree/release-1.3/examples/celery-rabbitmq
```
Comment (janetkuo) [inline _data/samples.yml]: This example was removed in 1.5 release because it no longer works.

File: docs/user-guide/ingress.md [hunk 3]
```diff
@@ -220,7 +220,7 @@ Note that there is a gap between TLS features supported by various Ingress contr
...
+It's also worth noting that even though health checks are not exposed directly through the Ingress, there exist parallel concepts in Kubernetes such as [readiness probes](/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/) which allow you to achieve the same end result. Please review the controller specific docs to see how they handle health checks ([nginx](https://github.com/kubernetes/contrib/blob/master/ingress/controllers/nginx/README.md), [GCE](https://github.com/kubernetes/contrib/blob/master/ingress/controllers/gce/README.md#health-checks)).
```
Comment (janetkuo) [inline docs/user-guide/ingress.md]: /docs/user-guide/production-pods.md was removed so changed to /docs/tasks/configure-pod-container/configure-liveness-readiness-probes/

**Comment by chenopis** [comment]
I checked all the links. Looks good.

**Pull request merged**